lotto/src/main/kotlin/domain/Lotto.kt

```diff
@@ -0,0 +1,45 @@
+package domain
+
+class WinningLotto(private val lotto: Lotto, private val bonus: Ball) {
+    init {
+        require(!lotto.contains(bonus)) { "보너스 숫자는 로또 번호와 중복될 수 없습니다" }
+    }
+
+    fun score(other: Lotto): Prize {
+        val hits = lotto.match(other)
+        val bonusHit = other.contains(bonus)
+        return Prize.match(hits, bonusHit)
+    }
+}
+
+class Lotto(strategy: LottoStrategy) {
+    private val balls: Set<Ball>
+
+    init {
+        balls = strategy.generateNumbers().map { Ball(it) }.toSet()
```

Comment:
```suggestion
class Lotto(strategy: LottoStrategy) {
    private val balls: Set<Ball> = strategy.generateNumbers().map { Ball(it) }.toSet()
```

필드 선언부로 옮겨도 괜찮을 것 같아요!

Reply:
😂 코잘알 후디 고맙습니다...